cpp/include/gar/graph.h

```diff
@@ -75,6 +75,19 @@ class Vertex {
   template <typename T>
   Result<T> property(const std::string& property) const;
 
+  /**
+   * @brief Return true if value at the property is valid (not null).
+   *
+   * @param property The property name.
+   * @return True if value at the property is valid, False otherwise.
+   */
+  inline bool IsValid(const std::string& property) const {
+    if (properties_.find(property) != properties_.end()) {
+      return properties_.at(property).has_value();
+    }
+    return true;
```

Comment:
I'm confused as to why the function returns true when the name is not found?

Reply:
Sorry, this is a very simple implementation. It doesn't validate the list_properties_. I wonder whether the return value should be true when the length of the list value is 0?

Reply:
> Sorry, this is a very simple implementation. It doesn't validate the list_properties_. I wonder whether the return value should be true when the length of the list value is 0?

Given that Arrow employs a distinct field to indicate null values, I think an empty array should not be equated with null.

Reply:
OK, I can resolve it.